src/models/sankey.js

```diff
@@ -0,0 +1,343 @@
+nv.models.sankey = function() {
+    'use strict';
+
+    // Sources:
+    // - https://bost.ocks.org/mike/sankey/
+    // - https://github.com/soxofaan/d3-plugin-captain-sankey
+
+    //============================================================
+    // Public Variables with Default Settings
+    //------------------------------------------------------------
+
+    var sankey = {},
+        nodeWidth = 24,
+        nodePadding = 8,
+        size = [1, 1],
+        nodes = [],
+        links = [],
+        sinksRight = true;
+
+    sankey.nodeWidth = function(_) {
+        if (!arguments.length) return nodeWidth;
+        nodeWidth = +_;
+        return sankey;
+    };
```

Comment:
It looks like the options on this file aren't using the chart._options format, which makes it so they can't be inherited automatically, please make this fit the same format as the rest of nvd3.